Content.Server/Medical/CrewMonitoring/CrewMonitoringConsoleSystem.cs

```diff
@@ -66,9 +67,9 @@ private void UpdateUserInterface(EntityUid uid, CrewMonitoringConsoleComponent?
                 return;
 
             // update all sensors info
-            var worldPos = _xform.GetWorldPosition(uid);
+            var xform = Transform(uid);
             var allSensors = component.ConnectedSensors.Values.ToList();
-            var uiState = new CrewMonitoringState(allSensors, worldPos, component.Snap, component.Precision);
+            var uiState = new CrewMonitoringState(allSensors, xform.WorldPosition, component.Snap, component.Precision);
```

Comment:
Needs updating?

Reply:
What do you mean? It works fine in game.